src/Components/Bookpage/BookDetails/bookdetails.css

```diff
@@ -8,7 +8,11 @@
     justify-content: center;
 }
 
-@media screen and (max-width: 990px) {
+.book-image img{
+    width: 40vw;
+}
+
+@media screen and (max-width: 450px) {
     .book-details {
         flex-direction: column;
         max-width: 80vw;
```

Comment:
Using max-width with a percentage/vw doesn't make sense because 80vw will never be more than 80vw. You either need a  static value here... or just width property

Reply:
I realise this was here before your contribution but didn't notice it before